Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1804,6 +1804,8 @@ private string GetVideoArguments(StreamState state, int startNumber, bool isEven
         {
             if (EncodingHelper.IsCopyCodec(codec)
                 && (state.VideoStream.VideoRangeType == VideoRangeType.DOVI
+                    || state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHDR10
+                    || state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHLG
```

Comment:
Some clients do not support `dvh1` FourCC but they support HDR10 & HLG in `hvc1`, such as the HLS.js used in non-Safari browsers. Please refer to the [code here](https://github.com/jellyfin/jellyfin/blob/fab3151679999bddf767f8c1e5e0826f58f63e4f/MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs#L2200-L2212) to set the appropriate video tag according to the VideoRangeType reported by the client.